package.json

```diff
@@ -1,13 +1,13 @@
 {
   "name": "edge-diagnostics-adapter",
-  "version": "0.5.0",
+  "version": "0.6.0-unsigned",
   "description": "Microsoft Edge Diagnostics Adapter",
-  "main": "out/src/edgeAdapter.js",
+  "main": "dist/edgeAdapter.js",
```

Comment:
that file doesn't exist, it should it should be like it was before `out/src/edgeAdapter`

Reply:
I though I reverted this changed :/